### PR TITLE
[IMP] l10n_br_fiscal: Hide page if company is not in Brazil

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Review by felipemotter — APPROVED** [review]

**Review by antoniospneto — APPROVED** [review]

**Review by rvalyi — CHANGES_REQUESTED** [review]
Hello @max3903 thanks for contributing. For the feature to be working as expected we should also change the attrs of the icms_regulation_id field (in the same XML file) to make it not required if the company is not Brazil. Because currently, if you create a company outside from Brazil, even if the fiscal tab is hidden, by default this field is still required while being invisible. See this print (were I left the company in Brazil to let the tab visible):

**Review by renatonlima — CHANGES_REQUESTED** [review]
Hello @max3903,

Can you change the actual required fields inside page Fiscal to false?

**Comment by felipemotter** [comment]
@renatonlima wouldn't It be better set required = false If It isn't a brazilian Company?

**Comment by renatonlima** [comment]
> wouldn't It be better set required = false If It isn't a brazilian Company?

That's exactly what I meant in last comment, to use the same domain in attrs to hide Fiscal page to implements required attrs in these required fields.

**Comment by max3903** [comment]
@sandipvyas-osi Can you make the suggested changes?

**Comment by max3903** [comment]
@renatonlima @rvalyi It should be good now.

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Review by antoniospneto — APPROVED** [review]

File: l10n_br_fiscal/views/product_product_view.xml [hunk 1]
```diff
@@ -85,7 +94,11 @@
                             </field>
                         </group>
                         <group>
-                            <field name="fiscal_genre_id" required="1" />
+                            <field
+                                name="fiscal_genre_id"
+                                required="1"
+                                groups="l10n_br_fiscal.group_manager"
+                            />
```
Comment (rvalyi) [inline l10n_br_fiscal/views/product_product_view.xml:101]: @renatonlima @marcelsavegnago I think this group is not quite right. How could we fix it?

Comment (max3903) [reply l10n_br_fiscal/views/product_product_view.xml:101]: @rvalyi Using `attrs`...

File: l10n_br_fiscal/models/document_fiscal_mixin_methods.py [hunk 2]
```diff
@@ -53,6 +53,10 @@ def _compute_amount(self):
                             values[field] += line[field.replace("amount_", "")]
             doc.update(values)
 
+    def _compute_country(self):
+        for record in self:
+            record.country_id = self.env.company.country_id.id
+
```
Comment (antoniospneto) [inline l10n_br_fiscal/models/document_fiscal_mixin_methods.py:69]: I don't think it's a good option to take the currently active company, the company that owns the document could be another.

**Pull request closed without merging**